back/engines/commercial/user_custom_fields/lib/user_custom_fields/core_ext/enumerable.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
```

Comment:
Is it a good idea to put an extension to a Ruby module deep down in our code? Shouldn't we put it in an initializer outside the engines?

Reply:
I had some hesitation. In the end, I decided to put it in the engine with the idea that we could always move it later to the core if other parts of the code starts using it. But if we were to move it now, I'd still rather keep as a library and `require` it only when needed.